intelmq/tests/bots/parsers/html_table/test_feodotracker.py

```diff
@@ -0,0 +1,63 @@
+import os
+import unittest
+
+import intelmq.lib.test as test
+from intelmq.bots.parsers.html_table.parser import HTMLTableParserBot
+from intelmq.lib import utils
+
+with open(os.path.join(os.path.dirname(__file__), 'feodotracker.html')) as handle:
+    REPORT_DATA = handle.read()
+REPORT_LINES = REPORT_DATA.splitlines()
+
+
+REPORT = {"__type": "Report",
+          "raw": utils.base64_encode(REPORT_DATA),
+          }
+
+
+line1 = '%s/%s' % (REPORT_LINES[94][:-14], REPORT_LINES[94][-14:])
+EVENT1 = {"raw": utils.base64_encode(line1.strip()),
+          "__type": "Event",
+          "time.source": "2021-05-10T14:56:05+00:00",
+          "source.ip": "94.177.255.18",
+          "source.as_name": "AS199883 ARUBACLOUDLTD-ASN",
+          "classification.type": "c2server",
+          "malware.name": "dridex",
+          "source.geolocation.cc": "GB",
+          "status": "Online",
+          }
+line2 = '%s/%s' % (REPORT_LINES[95][:-14], REPORT_LINES[95][-14:])
+EVENT2 = {"raw": utils.base64_encode(line2.strip()),
+          "__type": "Event",
+          "time.source": "2021-05-10T14:56:04+00:00",
+          "source.ip": "203.114.109.124",
+          "source.as_name": "AS131293 TOT-LLI-AS-AP TOT Public Company Limited",
+          "classification.type": "c2server",
+          "malware.name": "dridex",
+          "source.geolocation.cc": "TH",
+          "status": "Offline",
+          }
+
+
+@test.skip_exotic()
```

Comment:
Is the test exotic?

Reply:
The HTML Table parser requires `beautifulsoup4`